vault/activity_log.go

```diff
@@ -1736,6 +1736,9 @@ func modifyResponseMonths(months []*ResponseMonth, start time.Time, end time.Tim
 		return months
 	}
 	start = timeutil.StartOfMonth(start)
+	if timeutil.IsCurrentMonth(end, time.Now().UTC()) {
+		end = timeutil.StartOfMonth(end).AddDate(0, -1, 0)
```

Comment:
it looks like this is setting `end` to the start of last month. is that intended?

my confusion comes from "should end be the end of a month or the start of a month? i'd think end"

Reply:
`end` is a function input so do you mean we should assign `<new_variable> := end`, do the intermediate arithmetic on `new_variable`, and then assign `end = new_variable`?

Reply:
yeah something like that. it seems to me like we're changing the meaning of `end` throughout this code, so it might be a good candidate to use a more targeted new variable?

Reply:
Sounds good, I'll open a separate PR for it!

Reply:
your call - it's not a huge deal but i think it would improve clarity?